rest-server/src/models/job.js

```diff
@@ -94,11 +95,41 @@ class Job {
             next(job, new Error('job not found'));
           } else {
             if (framework.frameworkStatus) {
+              let jobState = "";
+              const frameworkState = framework.frameworkStatus.frameworkState;
+              if (
+                  frameworkState == "FRAMEWORK_WAITING" ||
```

Comment:
pls use single quotes rather than double quotes

Reply:
@abuccts : What policy shall we follow to choose from single quotes and double quotes?

Reply:
1. Current js files use single quotes, so please be **consistent**. Otherwise you have to change all other quotes.
2. You can check [node](https://github.com/jscs-dev/node-jscs/blob/master/presets/node-style-guide.json#L47), [google](https://github.com/jscs-dev/node-jscs/blob/master/presets/google.json#L21) and [fb](https://github.com/facebook/fbjs/blob/c69904a511b900266935168223063dd8772dfc40/packages/eslint-config-fbjs-opensource/index.js#L247) for other style guides.

Reply:
Good suggestion. I will make the change.